### PR TITLE
Fix cell edit showing comma decimal seperator in edge

### LINKED ISSUE #813: When starting Table Edit mode, the data is shown with , as separator
https://issuetracker.deltares.nl/browse/DWO-707

### PR DESCRIPTION
**Pull request by hvangeffen**
Closes #813

### CODE REVIEW

File: src/components/table/TableCellEdit.vue [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
   <div class="table-cell-editable">
     <input type="checkbox" @change="(event) => selectItem(event)" />
+    <!-- Edge does not respect lang="en-US" as such to not have ',' as decimal separator we need type="text" -->
     <input
       :ref="`${props.item.date}-${props.id}-value`"
       v-model.number="currentItem.y"
       class="table-cell-edit"
-      type="number"
+      type="text"
+      inputmode="numeric"
```
Comment (wkramer) [inline src/components/table/TableCellEdit.vue:10]: Should this not be `decimal` to show the decimal separator on the keyboard (on mobile)?

**Review by wkramer — CHANGES_REQUESTED** [review]

File: src/components/table/TableCellEdit.vue (hunk 1 above)
Comment (hvangeffen) [reply src/components/table/TableCellEdit.vue]: Updated

File: src/components/table/TableCellEdit.vue [hunk 2]
```diff
@@ -1,10 +1,12 @@
 <template>
   <div class="table-cell-editable">
+    <!-- Edge does not respect lang="en-US" as such to not have ',' as decimal separator we need type="text" -->
```
Comment (ceesvoesenek) [inline src/components/table/TableCellEdit.vue:3]: :(

File: src/components/table/TableCellEdit.vue [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
     <input
       :ref="`${props.item.date}-${props.id}-value`"
       v-model.number="currentItem.y"
       class="table-cell-edit"
-      type="number"
+      type="text"
+      inputmode="decimal"
```
Comment (ceesvoesenek) [inline src/components/table/TableCellEdit.vue:9]: Edge does respect `inputmode`?

**Review by ceesvoesenek — APPROVED** [review]

File: src/components/table/TableCellEdit.vue (hunk 3 above)
Comment (hvangeffen) [reply src/components/table/TableCellEdit.vue:9]: it respects the fact that it won't modify a string passed with a `.` to a `,` if the `type="text"`

**Review by wkramer — APPROVED** [review]

**Pull request merged**